compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -125,6 +125,12 @@ class Config(RepresentationMixin):
         start command for that single-user endpoint within this timeframe, the
         single-user endpoint is started back up again.
         Default: 30 seconds
+
+    amqp_port : int | None
+        Port to use for AMQP connections. Note that only 5671, 5672, and 443 are
+        supported by the Compute web services. If None, the port is assigned by the
+        services (typically 5672).
+        Default: None
```

Comment:
For reference, "typically" depends on `amqp` or `amqps`:

- `amqp://fqdn/vhost` &rarr; 5672
- `amqps://fqdn/vhost` &rarr; 5671
- `amqps://fqdn:12355/vhost` &rarr; 12355

This is defined in `/etc/services` in Linux land.  (I don't think macOS has the `amqps` in it's definition, though.)